dev-cpp/valijson/valijson-1.0.5.ebuild

```diff
@@ -17,6 +17,9 @@ KEYWORDS="~amd64 ~arm64 ~x86"
 # needs patching to use those.
 RESTRICT="test"
 
+# outdated inspector/CMakeLists.txt (bug #957461)
```

Comment:
outdated isn't right. It's okay if it's _unused_ (and the directory is never traversed into). If it's _outdated_, it has to be fixed.